testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/api/core/ModuleInfoTestCase.java

```diff
@@ -0,0 +1,173 @@
+            },
+            {
+                "dependency-name" => "ModuleDependency",
+                    "module-name" => "org.jboss.modules:main",
+                    "export-filter" => "Reject",
+                    "import-filter" => "multi-path filter {exclude children of \"META-INF/\", exclude equals \"META-INF\", default accept}",
+                    "optional" => false
+            }
+            ],
+            "local-loader-class" => undefined,
+                    "resource-loaders" => [
+            {
+                "type" => "org.jboss.modules.JarFileResourceLoader",
+                    "paths" => [
+                "",
+                        "org/jboss/logmanager",
+                        "META-INF/services",
+                        "org",
+                        "META-INF/maven/org.jboss.logmanager/jboss-logmanager",
+                        "org/jboss",
+                        "org/jboss/logmanager/errormanager",
+                        "org/jboss/logmanager/formatters",
+                        "META-INF",
+                        "org/jboss/logmanager/filters",
+                        "org/jboss/logmanager/config",
+                        "META-INF/maven",
+                        "org/jboss/logmanager/handlers",
+                        "META-INF/maven/org.jboss.logmanager"
+                ]
+            },
+            {
+                "type" => "org.jboss.modules.NativeLibraryResourceLoader",
+                    "paths" => undefined
+            }
+            ]
+        }*/
+
+        // load module.xml
+        ModuleIdentifier identifier = ModuleIdentifier.fromString(TARGET_MODULE_NAME + ":main");
+        Module module = loadModule(identifier);
+
+        // run module-info operation
+        ModelNode op = Util.createEmptyOperation("module-info", RESOURCE);
+        op.get(NAME).set(TARGET_MODULE_NAME);
+        ModelNode response = getModelControllerClient().execute(op);
+        ModelNode result = validateResponse(response);
+
+        // compare module name
+        Assert.assertEquals("Unexpected name", TARGET_MODULE_NAME + ":main", result.get(NAME).asString());
+
+        // compare dependencies
+        List<ModelNode> dependencies = result.get("dependencies").asList();
+        DependencySpec[] xmlDependencies = module.getDependencies();
+        Map<String, ModuleDependencySpec> moduleDependencies = new HashMap<>();
+        for (DependencySpec d : xmlDependencies) {
+            if (d instanceof ModuleDependencySpec) {
+                ModuleDependencySpec mds = (ModuleDependencySpec) d;
+                moduleDependencies.put(mds.getIdentifier().getName() + ":" + mds.getIdentifier().getSlot(), mds);
+            }
+        }
+        int foundDependencies = 0;
+        for (ModelNode d : dependencies) {
+            String dependencyName = d.get("module-name").asString();
+            Assert.assertNotNull("Expected dependency", moduleDependencies.get(dependencyName));
```

Comment:
This should probably just use `Assert.assertTrue("Expected dependency", moduleDependencies.containsKey(dependencyName));`.